features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/ManyConcurrentNotifyNoBackgroundThreads.m

```diff
@@ -2,7 +2,6 @@
 
 @interface BSG_KSCrash
 + (instancetype)sharedInstance;
-- (void)setSuspendThreadsForUserReported:(BOOL)suspend;
 @end
 
 @interface ManyConcurrentNotifyNoBackgroundThreads ()
```

Comment:
This scenario can be removed as it's now identical to `ManyConcurrentNotifyScenario`